build/filters.js

```diff
@@ -27,6 +27,7 @@ module.exports.all = [
 	'!**/node_modules/**',
 
 	// {{SQL CARBON EDIT}}
+	'!extensions/**/out*/**',
```

Comment:
This should probably be undone, I'm not seeing anything indicating that this list is the problem (create-universal-app doesn't even use these filters - it's only used in the hygiene script)
```suggestion
```